### PR TITLE
Fix Bilbao subway icon

### PR DESCRIPTION
**Pull request by matheusgomesms**
Since the last PRs to add Bilbao icons (#4211 and #4338), the correct names of station and entrances were not added, I suppose. This PR aims to fix this. Not sure if I added everything that is needed (adding a new subway icon is a bit cumbersome, it seems), hope it's good!

**PS.: Strings must be regenerated. Could you please do that? Thanks! :D**

CC @endim8 @starsep 

Signed-off-by: Matheus Gomes

### CODE REVIEW

File: data/types.txt [hunk 1]
```diff
@@ -1363,6 +1363,7 @@ amenity|recycling|container
 *railway|station|subway|bangkok
 *railway|station|subway|beijing
 *railway|station|subway|bengalore
+*railway|station|subway|bilbao
```
Comment (starsep) [inline data/types.txt:1366]: I think new type shouldn't be added in the middle - number of line here is id of type.
Maps from previous release wouldn't be backward compatible

AFAIK it's ok to replace `mapswithme` or add at the end

**Comment by vng** [comment]
You should add bilbao subway entrance type into mapcss-mapping.csv only.
Let say instead of current ```deprecated|deprecated;1502;x``` entry

classificator.txt, types.txt will be generated based on this csv file with tools/unix/generate_drules.sh tool

**Comment by matheusgomesms** [comment]
Thanks for all the info! I reverted those files, and added the last two lines on mapcss-mapping.csv.

Also, I kept the additions of data/strings/types_strings.txt, I wasn't sure if I had to add those lines or not.

**Review by vng — COMMENTED** [review]
UPD: Sorry, everything is ok.

**Review by vng — APPROVED** [review]

**Pull request merged**